src/common/procedure/src/error.rs

```diff
@@ -34,6 +34,9 @@ pub enum Error {
     #[snafu(display("Loader {} is already registered", name))]
     LoaderConflict { name: String, location: Location },
 
+    #[snafu(display("Procedure Manager is stopped"))]
+    ProcedureManagerNotStart { location: Location },
```

Comment:
nitpick: Shorten the variant name.
```suggestion
    ManagerNotStart { location: Location },
```